pkg/scheduler/framework/v1alpha1/interface.go

```diff
@@ -160,6 +163,17 @@ type FilterPlugin interface {
 	Filter(pc *PluginContext, pod *v1.Pod, nodeName string) *Status
 }
 
+// PostFilterPlugin is an interface for Post-filter plugin. Post-filter is an
+// informational extension point. Plugins will be called with a list of nodes
+// that passed the filtering phase. A plugin may use this data to update internal
+// state or to generate logs/metrics.
+type PostFilterPlugin interface {
+	Plugin
+	// PostFilter is called by the scheduling framework after a list of nodes
+	// passed the filtering phase.
```

Comment:
can you add to the comment what Statues are expected: "All post-filter plugins must return Success or the pod will be rejected."

Reply:
also, it is worth documenting what the NodeToStatusesMap represent here: "the set of filtered nodes and their filter status"